docs/contributing/translation/ui-messages.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Translating UI Messages
+---
+
+Most content that needs to be translated is in Markdown files in translation repos. However, Gatsbyjs.org also has several pieces of UI text that should be translated. To do that, we use a library called [Lingui](https://lingui.js.org/).
+
+## Message Translation Guide
+
+Unlike Markdown documents, UI messages are kept in the same monorepo as the gatsby source.
+
+1. Follow the instructions for [running the Gatsby website](/contributing/blog-and-website-contributions/#making-changes-to-the-website).
+2. Go to `www/` and run `yarn lingui:add-locale [lang code]` to generate a file for your language at `src/data/locales/[lang-code]/messages.po`.
```

Comment:
How do we ensure that people use the correct lang code?

Reply:
It'll show up in the pull request of the generated po file.